FeedAPIChallenge/Feed API/FeedImagesMapper.swift

```diff
@@ -0,0 +1,48 @@
+//
+//  FeedImagesMapper.swift
+//  FeedAPIChallenge
+//
+//  Created by Chen Yi-Wei on 2021/9/3.
+//  Copyright © 2021 Essential Developer Ltd. All rights reserved.
+//
+
+import Foundation
+
+internal struct FeedImagesMapper {
+	private struct Root: Decodable {
+		let items: [Item]
+
+		var feedImages: [FeedImage] {
+			items.map { $0.feedImage }
+		}
+	}
+
+	private struct Item: Decodable {
+		let id: UUID
+		let description: String?
+		let location: String?
+		let url: URL
+
+		private enum CodingKeys: String, CodingKey {
+			case id = "image_id"
+			case description = "image_desc"
+			case location = "image_loc"
+			case url = "image_url"
+		}
+
+		var feedImage: FeedImage {
+			FeedImage(id: id, description: description, location: location, url: url)
+		}
+	}
+
+	private static var validCode: Int { 200 }
+
+	internal static func map(_ data: Data, from response: HTTPURLResponse) -> FeedLoader.Result {
```

Comment:
The convention in this codebase is to omit the default access level keyword of `internal`, so it should be removed.
```suggestion
	static func map(_ data: Data, from response: HTTPURLResponse) -> FeedLoader.Result {
```